src/buildercore/fastly/vcl/journal-google-scholar.vcl

```diff
@@ -0,0 +1,5 @@
+if (req.http.User-Agent ~ "Googlebot/") {
+    set req.http.X-eLife-Google-Scholar-Metadata = "1";
+} else {
+    unset req.http.X-eLife-Google-Scholar-Metadata;
```

Comment:
what's the reason for explicitly unset rather than not taking an action? Preventing the header setting from the outside? (but it would be security through obscurity as the user agent can be set)

Reply:
Yep. Limits it to the only triggering mechanism. (That header being an internal detail.)

It we really cared we could limit it to Google's IP range, but I wasn't going to bother so its easily testable. (And isn't a problem if someone else triggers it, just reduces the caching rate.)